spec/asyncapi.md

```diff
@@ -689,63 +689,101 @@ bindings:
 
-Describes a publish or a subscribe operation. This provides a place to document how and why messages are sent and received.
+##### Operations Object Example
+
+```json
+{
+  "onUserSignUp": {
+    "summary": "Action to sign a user up.",
+    "description": "A longer description",
+    "channel": {
+      "$ref": "#/channels/userSignup"
+    },
+    "action": "send",
+    "tags": [
+      { "name": "user" },
+      { "name": "signup" },
+      { "name": "register" }
+    ],
+    "bindings": {
+      "amqp": {
+        "ack": false
+      }
+    },
+    "traits": [
+      { "$ref": "#/components/operationTraits/kafka" }
+    ]
+  }
+}
+```
+
+```yaml
+onUserSignUp:
+  summary: Action to sign a user up.
+  description: A longer description
+  channel:
+    $ref: '#/channels/userSignup'
+  action: send
+  tags:
+    - name: user
+    - name: signup
+    - name: register
+  bindings:
+    amqp:
+      ack: false
+  traits:
+    - $ref: '#/components/operationTraits/kafka'
+```
+
+
+
+
+#### <a name="operationObject"></a>Operation Object
 
-For example, an operation might describe a chat application use case where a user sends a text message to a group. A publish operation describes messages that are received by the chat application, whereas a subscribe operation describes messages that are sent by the chat application.
+Describes a `send` or a `receive` operation.
 
 ##### Fixed Fields
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="operationObjectOperationId"></a>operationId | `string` | Unique string used to identify the operation. The id MUST be unique among all operations described in the API. The operationId value is **case-sensitive**. Tools and libraries MAY use the operationId to uniquely identify an operation, therefore, it is RECOMMENDED to follow common programming naming conventions.
+<a name="operationObjectAction"></a>action | `string` | **Required**. Allowed values are `send` and `receive`. Use `send` when it's expected that the application will send a message to the given [`channel`](#operationObjectChannel), and `receive` when the application should expect receiving messages from the given [`channel`](#operationObjectChannel).
+<a name="operationObjectChannel"></a>channel | [Reference Object](#referenceObject) | **Required**. A `$ref` pointer to the definition of the channel in which this operation is performed. Please note the `channel` property value MUST be a [Reference Object](#referenceObject) and, therefore, MUST NOT contain a [Channel Object](#channelItemObject). However, it is RECOMMENDED that parsers (or other software) dereference this property for a better development experience.
```

Comment:
As per the whole discussion in https://github.com/asyncapi/spec/issues/618, the `channel` field would expect a plain string, being the id of a channel. I guess by ID we would refer to https://github.com/asyncapi/spec/pull/719.

However, in the same issue linked to that PR, https://github.com/asyncapi/spec/issues/663#issuecomment-1036133392, we do have a discussion around using `$ref` as you added here.

Just to be more aligned, Is there any other discussion on this?

Reply:
@smoya @derberg @dalelane Nobody argued against https://github.com/asyncapi/spec/issues/663#issuecomment-1158094050. Does it mean are you ok if we move forward with this solution?

Reply:
added comment in linked issue